javascript/app-discover/src/components/graph/CausalGraphExplorer.tsx

```diff
@@ -160,7 +160,7 @@ export const CausalGraphExplorer: React.FC<{
 
 	const legendHeight = useMemo((): any => {
 		return legendRef?.current?.clientHeight ?? 0
-	}, [legendRef?.current?.clientHeight])
+	}, [])
```

Comment:
Nice catch! Since this `useMemo` only contains a single ref and simple logic, there might be no need for it to be memorized, I just removed the useMemo from here.